ament_pep257/ament_pep257/main.py

```diff
@@ -45,11 +46,20 @@ def main(argv=sys.argv[1:]):
     parser.add_argument(
         '--ignore',
         nargs='*',
-        default=[
-            'D100', 'D101', 'D102', 'D103', 'D104', 'D105', 'D106', 'D107',
-            'D203', 'D212', 'D404',
-        ],
-        help='The pep257 categories to ignore')
+        default=[],
+        help='The pep257 error codes to ignore.')
+    parser.add_argument(
+        '--select',
+        nargs='*',
+        default=[],
+        help='The pep257 error codes to check.'
+    )
+    parser.add_argument(
+        '--allow-undocumented',
```

Comment:
I kinda feel like this should be `action='store_true'`, but that might interfere with the backward compatibility check. What do you think?

Reply:
Yeah, there's no way to differentiate leaving out the argument from leaving it false in that case.

Reply:
Since the current behavior is contrary to pep257, we need a transition before adding this potentially breaking change.